woocommerce/woocommerce
# Customer Profiles: copy billing address to shipping address

See #14969 - the issue was about adding a checkbox for if the shipping address is different than billing, but I thought maybe "copy from billing" to fill in the shipping address would work better. This PR adds a new "field" to the form which is a copy button:

<img width="590" alt="screen shot 2017-05-15 at 5 12 57 pm" src="https://cloud.githubusercontent.com/assets/541093/26081505/13e71e4e-3999-11e7-8cf2-cba21b0df934.png">

Once clicked, it just pulls down all the matching billing address fields into shipping. The issue also mentioned pre-filling the fields with the information we have from the core WP user fields, which this PR also does for first name, last name, and email.

**To test:**

- Edit a user
- Change the billing information
- Click "Copy"
- See that the shipping address has also been updated

**To test the pre-filled fields:**

- Edit a user
- Delete billing first & last name
- Reload the page, these fields should be pre-filled with the user's first & last name from the Name section.
